### PR TITLE
update/ PyTorch (2.0.0 -> 2.0.1)

### PR DESCRIPTION
**Pull request by miolab**
# About

- [Update] PyTorch (2.0.0 -> 2.0.1)
- [Update] Sample code (handwritten digit recognition)
- [Fix] Exec `poetry add` __numpy__ and __pytorch__ before `poetry install` other libraries for the purpose of avoiding timeout errors.

### PyTorch 2.0

- https://pytorch.org/get-started/pytorch-2.0/
- https://github.com/miolab/jupyterlab_poetry/pull/63/files

### Others
- poetry add with --index-url option
  - https://github.com/miolab/jupyterlab_poetry/pull/63/files
  - https://github.com/python-poetry/poetry/pull/8246/files

### CODE REVIEW

**Comment by miolab** [comment]
Training loss over epochs results plot

image

**Comment by miolab** [comment]
We have confirmed that `poetry install` does __NOT cause a time error__ and that pytorch works.
